octave/tm/tmrepl.m

```diff
@@ -36,17 +36,9 @@ function tmrepl()
     eval (__r, "tmlasterr");
 
     if disp_ans
-      global TM_OCTAVE_PLOT_DIGEST;
-      
-      updated_digest= hash ("md5", serialize (get(gcf())));
```

Comment:
This is the only usage for `octave/kernel/serialize.m`. If we remove `serialize`, `octave/kernel/serialize.m` can also be removed.